src/main/java/org/jboss/arquillian/extension/jacoco/client/filter/AndFilter.java

```diff
@@ -16,13 +16,18 @@
     @Override
     public boolean include(T object)
     {
+        boolean include = false;
         for (Filter<T> f : filters)
         {
             if (!f.include(object))
             {
+                if (f.toString().contains("IncludeRegExpPaths") && include) {
```

Comment:
Yes, [this test](https://github.com/arquillian/arquillian-extension-jacoco/pull/31/commits/b3a89f775131bb67e4f2a157b45555607dc59183#diff-3000a9437ea469d92e6b9063dc369723L127).